archive/j/java/BinarySearch.java

```diff
@@ -0,0 +1,44 @@
+import java.io.*;
```

Comment:
Looks like this is a carry over file from #1635. You may need to remove it on this branch.